commonv5/ts/sonarqube/Analysis.ts

```diff
@@ -133,18 +147,13 @@ export default class Analysis {
   }
 
   private getQualityGateColor() {
-    switch (this.status) {
-      case "OK":
-        return "#00aa00";
-      case "WARN":
-        return "#ed7d20";
-      case "ERROR":
-        return "#d4333f";
-      case "NONE":
-        return "#b4b4b4";
-      default:
-        return "#b4b4b4";
-    }
+    const colors = {
+      OK: "#6EB712",
```

Comment:
How did you know what color to use there? I do not see it specified on figma?

Reply:
I looked at the colors in [DS](https://www.figma.com/file/kuN046eNTwGnRwIsiP6Cjf/ProductDesignSystem?node-id=14678%3A5686&mode=dev) and background colors made sense to me and I double checked them with Simo as well